tests/test_generic/tests/test_industry_mail_behavior.py

```diff
@@ -0,0 +1,33 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo.tests import get_db_name, tagged
+from .industry_case import IndustryCase
+
+_logger = logging.getLogger(__name__)
+
+
+@tagged('post_install', '-at_install')
+class IndustryMailBehaviorTestCase(IndustryCase):
+    def test_user_notification_type(self):
+        for _ in self.installed_industries:
+            db_name = get_db_name()
+            if db_name.endswith('imported_no_demo'):
```

Comment:
Better an early return for readability (and split this test into two separate ones)
```suggestion
            if db_name.endswith('imported_with_demo'):
                return
```